src/components/tag/_tag.scss

```diff
@@ -20,5 +19,11 @@
 
     text-decoration: none;
     text-transform: uppercase;
+
+    &--inactive {
+      background-color: $govuk-grey-2;
+
```

Comment:
There are two consecutive empty lines here resulting in a  `No trailing whitespace allowed` warning.

Reply:
Where are you seeing that warning? Should it be failing CI?